hdr_html_canvas_element.md

```diff
@@ -112,77 +114,100 @@ Update `PredefinedColorSpace` to include the following new color spaces.
   }
 ```
 
+#### extended-linear-srgb
+
+The component signals are mapped to red, green and blue tristimulus values according to the following:
+
+* Red primary: `(0.640, 0.330)`
+* Green chromaticity: `(0.300, 0.600)`
+* Blue chromaticity: `(0.150, 0.060)`
+* White chromaticity: `(0.3127, 0.3290)`
+* Transfer function:
+
+```
+   E = | E' / 12.92, if abs(E') ≤ 0.04045
+       | ((E' + 0.055) / 1.055)^2.4, otherwise
+
+       with E' ∈ ℝ
+```
+
+#### extended-srgb
+
+The component signals are mapped to red, green and blue tristimulus values according to the following:
+
+* Red primary: `(0.640, 0.330)`
+* Green chromaticity: `(0.300, 0.600)`
+* Blue chromaticity: `(0.150, 0.060)`
+* White chromaticity: `(0.3127, 0.3290)`
+* Transfer function: `E = E', with E' ∈ ℝ`
+
+#### rec2100-hlg
+
+The component signals are mapped to red, green and blue tristimulus values according to the Hybrid Log-Gamma (HLG) system specified in Rec. ITU-R BT.2100.
+
+#### rec2100-pq
+
+The component signals are mapped to red, green and blue tristimulus values according to the PQ system system specified in Rec. ITU-R BT.2100.
+
 ### Conversion between color spaces
 
-All `PredefinedColorSpace` are defined by how they are converted to XYZD50 under relative colorimetric intent.
+#### General
 
-Converting from XYZD50 to a `PredefinedColorSpace` is done by performing the inverse of the conversion to XYZD50.
+This section specifies the color space conversion when compositing element with color space A onto a canvas element with color space B.
 
-Color values in XYZD50 may assume any real value (including values less than zero and greater than one).
+These conversions are expressed using a connection color space with the system colorimetry specified in Rec. ITU-R BT.2100:
```

Comment:
The HTML spec [uses XYZD50](https://www.w3.org/TR/css-color-4/#predefined-to-lab), so I'd just stick with that pattern.

At a minimum the connection color space should be one of the [CSS Predefined Color Spaces](https://www.w3.org/TR/css-color-4/#predefined) (or one of the color spaces we plan to add to that set), so that we don't have to re-define it here.

Reply:
If the connection space is used merely for mathematical expediency, why use a more complicated mathematical expression and a white point which is not relevant for HDR color systems?